mentii/mentii
# Add book

Adds the "Add book" functionality for an admin

**Significant Changes**
1. Added creation of books table (see LocalDB and BuildScript repo)
2. Unit tests in Backend
3. Form to add new book in Frontend
4. Backend implementation to handle request and form model

**How to Test**
1. Login
2. Confirm dropdown in **Admin Portal** can select **Create New Book**
3. Go to admin page and confirm you can select between **Change User Role** and **Add New Book**
4. Fill out **Title** and **Description** and confirm **Create Book** is enabled
5. Add a new chapter and confirm **Create Book** is disabled
6. Fill out **Chapter Title** field and confirm **Create Book** is enabled
7. Repeat for adding/removing Chapters, Sections, or Problems
8. After pressing **Create Book** confirm book you created is in your LocalDB

**Notes**
:watermelon:
